.github/workflows/ci.yml

```diff
@@ -90,3 +90,38 @@ jobs:
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           unchecked: '1,5069'
+
+  markdownlint:
+    name: Markdown Linter
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout EIP Repository
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
+
+      - name: Get Changed Files
+        id: changed-files
+        uses: tj-actions/changed-files@6c44eb8294bb9c93d6118427f4ff8404b695e1d7
+
+      - name: Delete Unmodified Files
+        run: |
+          cd $GITHUB_WORKSPACE
+          mkdir all
+          shopt -s extglob dotglob
+          mv !(all) all
+          shopt -u dotglob
+          mkdir unchanged
+          cd all
+          for FILE in ${{ steps.changed-files.outputs.all_changed_files }}; do
+            [ -f ${FILE} ] && mkdir -p $(dirname "../unchanged/${FILE}")
+            [ -f ${FILE} ] && cp ${FILE} ../unchanged/${FILE}
+          done
+          cd ..
+          rm -rf all
+          cp -r unchanged/* .
+          cd ..
+          rm -rf unchanged
```

Comment:
If we are going to be using this for multiple workflows, I would like to see it formalized into a self contained action, or some other piece of reusable code (note: having reusable code that lives outside of a workflow is a security risk, so an action is preferred).